blackboard/gui/result_pane.py

```diff
@@ -0,0 +1,28 @@
+import gi
+gi.require_version('Gtk', '3.0')
+
+from gi.repository import Gtk, Gio, Gdk  # noqa
+
+
+class result_pane(Gtk.Grid):
+    def __init__(self):
+        Gtk.Grid.__init__(self)
+        self.lb_list = []
+        self.__add_label(0)
+
+    def __add_label(self, index):
+        lb = Gtk.Label(str(index))
+        self.lb_list.insert(index, lb)
+        if (len(self.lb_list) - 1) > index:
+            self.insert_row(index)
+        self.attach(lb, 0, index, 1, 1)
+        lb.show()
+
+    def capture_pressed_key(self, txt_op, value):
```

Comment:
This doesn't make me sense
Why a method called `capture_pressed_key` calculate an index and add a label?
`capture_pressed_key` is very generic name maybe this method should be called `__add_label_at_cursor` and `__add_label` must be called `__add_label_at`. Another method called `__get_cursor_position`should be implemented with the txt_op as parameter and at the end, this method should return the index